fe/fe-core/src/main/java/com/starrocks/server/CatalogMgr.java

```diff
@@ -132,6 +133,9 @@ public void replayCreateCatalog(Catalog catalog) throws DdlException {
         if (Strings.isNullOrEmpty(type)) {
             throw new DdlException("Missing properties 'type'");
         }
+        if (Config.ignore_unknown_connector_type && !CreateCatalogStmt.supportedCatalog.contains(type)) {
```

Comment:
The config param is not required. If encountered an unknown type, just return and print a warning log. Too many config params will mislead users.